ripple/src/main/webapp/script.js

```diff
@@ -74,6 +74,24 @@ function selectFile(fileId) {
   document.getElementById(fileId).click();
 }
 
+function viewAllPostComments() {
+  alert("Fetch all comments for this post!");
+}
+
+/* Backend for search functionality */
+
+/* Takes navbar search input, stores in session storage. */
+function searchInput(keyPress) {
+    if (keyPress.keyCode === ENTER_KEY) {
+      var searchString = document.getElementById('search-bar').value;
+      console.log(searchString);
+      localStorage.setItem('searchString', searchString);
+      localStorage.setItem('galleryPageName', searchString);
+      window.location.assign("businessgallery.html");
+    } else {
+        return false;
+    }
+
```

Comment:
Is the closing bracket missing for searchInput()? Also indentation seems off for this method.

Reply:
I must have messed this up when merging! Just fixed.